docs/basic_training/intro.md

```diff
@@ -142,7 +143,24 @@ workflow { // (18)!
 21. The final output (in the `results_ch` channel) is printed to screen using the `view` operator (appended onto the channel name).
 22. End of the workflow scope.
 
-The use of the operator `.flatten()` here is to split the two files into two separate items to be put through the next process (else they would be treated as a single element).
+This pipeline takes `params.greeting`, which defaults to the string `Hello world!`, and splits it into indiviudual words in the `SPLITLETTERS` process. Each word is written to a separate file, named `chunk_1`, `chunk_2` and so on. These files are picked up as the process `output`.
```

Comment:
```suggestion
This pipeline takes `params.greeting`, which defaults to the string `Hello world!`, and splits it into indiviudual words in the `SPLITLETTERS` process. Each word is written to a separate file, named `chunk_aa`, `chunk_ab`,  `chunk_ac`and so on. These files are picked up as the process `output`.
```

Reply:
Ah oops, good spot 👍🏻 

Do you think I should make the Python version have identical behaviour?

Reply:
Meh, can't think of a nice syntax to do it that doesn't complicate the example more, so will leave the Python as it is I think.

Reply:
Keep it simple - I'd leave it as is